# ====== DO NOT MODIFY! AUTOGENERATED FROM AN ACTION DEFINITION ======

FindBookActionGoal action_goal
FindBookActionResult action_result
FindBookActionFeedback action_feedback
